website/docs/r/compute_instance_template.html.markdown

```diff
@@ -287,6 +287,11 @@ The `disk` block supports:
 * `type` - (Optional) The type of GCE disk, can be either `"SCRATCH"` or
     `"PERSISTENT"`.
 
+* `labels` - (Optional) A set of key/value label pairs to assign to the disk.
+
+    If you are creating a new disk, these labels are added to the disk.
+    If you are attaching an existing disk, these labels are ignored.
```

Comment:
```suggestion
    You cannot use labels if you are attaching an existing disk using `source`.
```
quick clarification since once my linked PR is merged it will fail instead of getting silently ignored as it does right now.